test/Dashboard.EndToEndTests/Tests/ArgumentsDisplay/QueueArgumentsDisplayFunctions.cs

```diff
@@ -59,17 +59,17 @@ public class QueueArgumentsDisplayFunctions
             done = "x";
         }
 
-        public static void ICollector(
+        public static async Task ICollector(
             [QueueTrigger("queue-icollector-in")] string input,
-            [Queue("queue-icollector-out")] ICollector<string> output,
-            [Queue(DoneNotificationFunction.DoneQueueName)] out string done)
+            [Queue("queue-icollector-out")] IAsyncCollector<string> output,
```

Comment:
Yep, but using ICollector deadlocks, it's the same 'sync over async' stuff discussed with the core sdk